dts/bindings/led/nxp,pca9634.yaml

```diff
@@ -0,0 +1,10 @@
+description: NXP PCA9634 LED
+
+compatible: "nxp,pca9634"
+
+include: i2c-device.yaml
+
+properties:
+  external-driver:
+    type: boolean
+    required: false
```

Comment:
Please add a `description:` with an explanation of what setting this to `true` means.

Reply:
Done.